.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_HOST = "http://localhost:7072"
```

Comment:
please remove this file

Reply:
And how will the others have the api??

Reply:
you can pass the env internally to the other devs

Reply:
@rodrigo-baldiviezo-jalafund please add an .env.example file this will help other devs to know which variables they need to run the application
```
REACT_APP_API_HOST=
...here you'll add other variables in future
```

Reply:
done!